adapters/ix/ix.go

```diff
@@ -417,6 +420,28 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 				}
 			}
 
+			var bidNative1_1 *Native11Wrapper
+			if bidType == openrtb_ext.BidTypeNative {
+				unmarshalExtErr := json.Unmarshal([]byte(bid.AdM), &bidNative1_1)
+				if unmarshalExtErr == nil && len(bidNative1_1.Native.EventTrackers) > 0 {
+					mergeNativeImpTrackers(&bidNative1_1.Native)
+					if json, marshalErr := json.Marshal(bidNative1_1); marshalErr == nil {
+						bid.AdM = string(json)
+					}
+				}
+			}
+
+			var bidNative1_2 *native1response.Response
+			if bidType == openrtb_ext.BidTypeNative {
+				unmarshalExtErr := json.Unmarshal([]byte(bid.AdM), &bidNative1_2)
```

Comment:
Same as above: This variable name could be better since you aren't unmarshallng an ext field. Perhaps `unmarshalErr` or `err` would work.

Reply:
Done.